aspnetcore/mvc/models/model-binding.md

```diff
@@ -966,6 +966,8 @@ For targets that are collections of simple types, model binding looks for matche
   ```
   [a]=1050&[b]=2000&index=a&index=b
   ```
+  > [!WARNING] 
```

Comment:
Maybe we could remove the warning and simply state it? It's not really a error case.

Reply:
@pranavkm, we could if you prefer but I think that effect of using the index parameter cause enough disturbance, since the list will be populated with an item that could potentially break the user's code.

Reply:
Can you check [my commit](https://github.com/dotnet/AspNetCore.Docs/pull/24175/commits/86fed85d8b11e542770b09466ae74d0d681f970c)?

Reply:
I like that, it is pretty clear. Thanks both for the help